src/04-modality-specific-files/29-motion.md

```diff
@@ -0,0 +1,252 @@
+# Motion
+
+Support for Motion was developed as a
+[BIDS Extension Proposal](../07-extensions.md#bids-extension-proposals).
+Please see [Citing BIDS](../01-introduction.md#citing-bids)
+on how to appropriately credit this extension when referring to it in the
+context of the academic literature.
+
+Several [example Motion datasets](https://github.com/bids-standard/bids-examples)
```

Comment:
I guess either URL should be more specific, or a list of example dataset names in there should be given

Reply:
We will change this as soon as the example datasets are merged into the main branch